### DISCUSSION TITLE
friendlyarm_nanopi-r5c设备固件immortalwrt-23.05.2异常。

### DISCUSSION THREAD

**Discussion by cqmr** [immortalwrt/immortalwrt#1349] [Q&A]
### Describe the bug

friendlyarm_nanopi-r5c设备装配有rtl8822ce无线网卡，成功刷入固件immortalwrt-23.05.2-rockchip-armv8-friendlyarm_nanopi-r5c-ext4-sysupgrade，重启设备，成功完成启动后，在we设置中，网络菜单下没有无线网卡的配置菜单，且R5C设备的LED无线网卡灯不亮，无法判断8822CE无线网卡是否正常工作【在friendlywrt22.03固件下，8822CE无线网卡一切正常(连接和LED灯均正常),已排除硬件故障】。如下图
![imm-err-23 05 2](https://github.com/immortalwrt/immortalwrt/assets/169730908/84fdf9e0-966b-42ed-bf32-d70df999796f)
进入终端使用dmesg命令，查看信息如下。
[ImmortalWrt-23.05.2-dmesg.txt](https://github.com/user-attachments/files/16032531/ImmortalWrt-23.05.2-dmesg.txt)。
不懂分析方法，只能提交大神们研究。

### ImmortalWrt version

r27625-416c8c5c91

### ImmortalWrt release

23.05.2

### ImmortalWrt target/subtarget

rockchip/armv8

### Device

FriendlyElec NanoPi R5C

### Image kind

Official downloaded image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by 1715173329** [comment]
可能相关 https://github.com/immortalwrt/immortalwrt/issues/1132#issuecomment-2163766615
